apps/shopifyneworder/shopify_new_order.star

```diff
@@ -0,0 +1,267 @@
+    background_color = api_config.get("backgroundColor")
+    logo = api_config.get("logo", IMAGE_RECENT_ORDERS)
+
+    return render.Root(
+        child = render.Column(
+            expanded = True,
+            children = [
+                render_header_row(base64.decode(logo), "new orders\this month", text_color, background_color),
+                render.Box(
+                    width = 64,
+                    height = 1,
+                    color = COLOR_WHITE,
+                ),
+                render.Column(
+                    expanded = True,
+                    children = [
+                        render_marquee(orders),
+                    ],
+                ),
+            ],
+        ),
+    )
+
+def render_marquee(orders):
+    order_views = []
+    for order in orders:
+        order_views.append(
+            render.Padding(
+                child = render.Text(
+                    content = order["line_item_count"],
+                    font = FONT_TOM_THUMB,
+                    color = COLOR_WHITE,
+                ),
+                pad = (7, 0, 4, 0),
+            ),
+        )
+        order_views.append(
+            render.Text(
+                content = order["current_total_price"],
+                font = FONT_TOM_THUMB,
+                color = COLOR_ALOE,
+            ),
+        )
+    return render.Stack(
+        children = [
+            render_gradient(64, 32, "000000", "1238BF"),
+            render.Box(
+                child = render.Marquee(
+                    width = 64,
+                    offset_start = 8,
+                    child = render.Row(
+                        children = order_views,
+                        expanded = True,
+                    ),
+                ),
+                height = 10,
+            ),
+        ],
+    )
+
+def render_header_row(image, title, text_color, background_color):
+    return render.Box(
+        child = render.Row(
+            children = [
+                render.Box(
+                    width = 20,
+                    child = render.Image(
+                        src = image,
+                    ),
+                ),
+                render.WrappedText(
+                    content = "new orders\nthis month",
+                    font = FONT_TOM_THUMB,
+                    align = "center",
+                    color = text_color,
+                ),
+            ],
+            main_align = "space_around",
+            cross_align = "center",
+            expanded = True,
+        ),
+        height = 22,
+        color = background_color,
+    )
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
+                id = "counterId",
+                name = "Counter ID",
+                desc = "Unique ID of the counter set up in the Counter app for Shopify",
+                icon = "shopify",
+            ),
+            schema.Text(
```

Comment:
There's going to be some usability issues here for current Tidbyt users - though we don't have a color picker at the moment. The only alternative would be a dropdown